src/test/ui/const-generics/outer-lifetime-in-const-generic-default.rs

```diff
@@ -0,0 +1,10 @@
+struct Foo<
+    'a,
```

Comment:
haha rustfmt is so weird, i swear i didn't do this

Reply:
^^